table_foosball/table_foosball_01.rb

```diff
@@ -0,0 +1,141 @@
+require "securerandom"
+
+class Player
+  def initialize name
+    @name = name
+    @games = [] # what are all the games this player has been involved in?
```

Comment:
I don't think we should store the games on the Player, instead I think we should either create a new class called `GameManager` that stores them, or create class methods on `Game` that does it.

E.g.

```ruby
class Game
  def self.all
    @games
  end

  def self.create(game)
    @games << game
  end

  # instance methods go here
end
```

Then the methods below instead will look like:

```
def games
  Game.all.select { |game| game.players.include?(self) }
end

def won_games
  games.select {|g| g.winner.include?(self) }
end
```